be/src/exec/parquet_writer.h

```diff
@@ -63,14 +63,35 @@ class ParquetOutputStream : public arrow::io::OutputStream {
     int64_t _written_len = 0;
 };
 
+class ParquetBuildHelper {
+public:
+    static void build_schema_repetition_type(
+            parquet::Repetition::type& parquet_repetition_type,
+            const TParquetRepetitionType::type& column_repetition_type);
+
+    static void build_schema_data_type(parquet::Type::type& parquet_data_type,
+                                       const TParquetDataType::type& column_data_type);
+
+    static void build_compression_type(parquet::WriterProperties::Builder& builder,
+                                       const TParquetCompressionType::type& compression_type);
+
+    static void build_version(parquet::WriterProperties::Builder& builder,
+                              const TParquetVersion::type& parquet_version);
+};
+
 // a wrapper of parquet output stream
 class ParquetWriterWrapper {
 public:
-    ParquetWriterWrapper(FileWriter* file_writer, const std::vector<ExprContext*>& output_expr_ctxs,
-                         const std::map<std::string, std::string>& properties,
-                         const std::vector<std::vector<std::string>>& schema,
-                         bool output_object_data);
-    virtual ~ParquetWriterWrapper();
+    ParquetWriterWrapper(doris::FileWriter* file_writer,
+                         const std::vector<ExprContext*>& output_vexpr_ctxs,
+                         const std::vector<TParquetRepetitionType::type>& schemas_repetition_type,
+                         const std::vector<TParquetDataType::type>& schemas_data_type,
+                         const std::vector<std::string>& schemas_column_name,
+                         const TParquetCompressionType::type& compression_type,
+                         const bool& parquet_disable_dictionary,
+                         const TParquetVersion::type& parquet_version, bool output_object_data);
+
+    virtual ~ParquetWriterWrapper() = default;
```

Comment:
if `ParquetWriterWrapper` do not have subclass, this function do need to be virtual